package.json

```diff
@@ -39,7 +39,7 @@
   "homepage": "https://github.com/mitodl/course-search-utils#readme",
   "dependencies": {
     "@mitodl/open-api-axios": "2024.9.16",
-    "@remixicon/react": "^4.2.0",
+    "@remixicon/react": "4.0.1",
```

Comment:
This does mean that some of our projects will get duplicated versions of `@remixicon/react`. E.g., Learn will install 4.2 itself, and get 4.01 from this package. Not really a big deal, since the contribution to bundle from 5 svg components used here is going to be very small.

Or we could just use 4.2 and demand upgrades to 18. (The current main-branch version of course-search-utils is v3; nothing is using v3 except Learn; the only project using v2 is OCW, which would be fairly simple to upgrade to 18.)

Apparently course-search-utils also specifies react 16 as a peer dep, so the peer deps are already incorrect a bit, btw =/

Reply:
If it's likely other projects will need upgrades to course-search-utils (expected to accommodate any breaking changes to the API), it could block them if they had any issues upgrading v16 to v18 (though I think React 16->18 changes are backwards compatible, even if not supported). Thinking that's excessive for a couple of icons, though yes - projects will end up with the 2 versions installed together.